test/position-managers/PositionManager.t.sol

```diff
@@ -61,6 +61,13 @@ contract PositionManagerTest is Test, PosmTestSetup, LiquidityFuzzers {
         approvePosmFor(alice);
     }
 
+    function test_posm_initcodeHash() public {
+        vm.snapshotValue(
+            "position manager initcode hash (without constructor params, as uint256)",
+            uint256(keccak256(abi.encodePacked(vm.getCode("PositionManager.sol:PositionManager"))))
```

Comment:
can you add one for v4router too please?

Reply:
i think this function should be in `PositionManager.gas.t.sol`

its more consistent with how we're doing it in core, and will add the snapshot to `PosMGasTest.json`